zipline/finance/performance/period.py

```diff
@@ -90,11 +91,49 @@
 from zipline.utils.serialization_utils import (
     VERSION_LABEL
 )
+from zipline.finance.performance.position_tracker import calc_position_stats
 
 log = logbook.Logger('Performance')
 TRADE_TYPE = zp.DATASOURCE_TYPE.TRADE
 
 
+PeriodStats = namedtuple('PeriodStats',
+                         ['net_liquidation',
+                          'gross_leverage',
+                          'net_leverage'])
+
+
+def calc_net_liquidation(ending_cash, long_value, short_value):
+    return ending_cash + long_value + short_value
+
+
+def calc_gross_leverage(gross_exposure, net_liq):
+    if net_liq != 0:
+        return gross_exposure / net_liq
+
+    return np.inf
+
+
+def calc_net_leverage(net_exposure, net_liq):
```

Comment:
Can we just call this function `calc_leverage` and call it twice with different arguments.